.travis.yml

```diff
@@ -13,6 +13,11 @@ matrix:
     - python: 3.9-dev
       env: TOXENV=py39
 
+addons:
+  apt:
+    packages:
+      - libpulse-mainloop-glib0
```

Comment:
What is this needed for?

Reply:
`stubtest` actually processes (runs?) the real package and Qt multimedia stuff had issues.

Reply:
```suggestion
      # Required to stubtest QtMultimedia
      - libpulse-mainloop-glib0
```